paragon/_overrides.scss

```diff
@@ -0,0 +1,149 @@
+@import "../common/overrides";
+
+header {
+  // Set the header background color
+  background-color: $header-bg;
+
+  // Set the header for desktop
+  &.site-header-desktop {
+    background-color: $header-bg;
+  }
+
+  @if variable-exists(customize-header-border) {
+    // Set the top and bottom border of the header
+    border-top: $header-border-top-width solid $header-border-top-color;
+    border-bottom: $header-border-bottom-width solid $header-border-bottom-color;
+    // The default height is 3.75rem, to that we add the additional border size, otherwise
+    // it breaks the UI.
+    height: calc(3.75rem + #{$header-border-top-width + $header-border-bottom-width});
+  }
+  // Set the color of text in the header - course about page.
+  @if variable-exists(main-nav-color) {
+    span {
+      color: $main-nav-color;
+    }
+  }
+
+  .nav-links {
+    @if variable-exists(main-nav-link-color) {
+      // Set the color of the links in the dashboard
+      .main .nav-tab a,
+      .secondary .nav-item a,
+      .secondary .nav-item a span,
+      .toggle-user-dropdown span {
+        color: $main-nav-link-color !important;
+      }
+    }
+
+    @if variable-exists(main-nav-item-border-bottom-color) {
+      // Set the bottom border color of the links in the dashboard like 'Courses', 'Programs', 'Profile' etc.
+      .main .nav-tab a {
+        &.active {
+          border-bottom-color: $main-nav-item-border-bottom-color !important;
+          border-bottom-style: solid !important;
+        }
+      }
+    }
+
+    @if variable-exists(main-nav-item-hover-border-bottom-color) {
+      .main .nav-tab a {
+        &:not(.active):hover {
+          border-bottom-color: $main-nav-item-hover-border-bottom-color !important;
+          border-bottom-style: solid !important;
+        }
+      }
+    }
+
+    @if variable-exists(user-dropdown-color) {
+      // Customize the color of the links in the user dropdown menu
+      .secondary .dropdown-user-menu .dropdown-item a {
```

Comment:
It doesn't seem to be matching the current dropdown. Currently, these items are something along `nav.secondary-menu-container .dropdown-menu .dropdown-item`.

Reply:
Updated.